generator/src/main/java/com/scottlogic/deg/generator/walker/reductive/ReductiveIterationVisualiser.java

```diff
@@ -73,7 +72,6 @@ private void createVisualiseDirectoryIfAbsent() throws IOException {
                 " directory needs to exist.\n" + this.visualiseDirectoryPath.toString());
         }
 
-        FileSystemProvider provider = FileSystems.getDefault().provider();
-        provider.createDirectory(this.visualiseDirectoryPath);
+        new FileUtils().createDirectories(this.visualiseDirectoryPath);
```

Comment:
Changed from using `FileSystemProvider` to `Files` as `FileSystemProvider` is used to create new FileSystems. `Files.createDirectories()` is the recommended way to create directories

Reply:
Are you able to inject FileUtils rather than creating it here?

Reply:
good point, yes. updated.